scss/_variables.scss

```diff
@@ -114,23 +114,27 @@ $enable-print-styles:       true !default;
 // variables. Mostly focused on spacing.
 // You can add more entries to the $spacers map, should you need more variation.
 
+// stylelint-disable
```

Comment:
Can't we be more explicit with what we disable here?

Reply:
As in case for other empty maps. Stylelint does not allow empty maps ex:
```sass
// stylelint-disable
$grays: () !default;
// ...
$colors: () !default;
// ...
$theme-colors: () !default;
// ...
// stylelint-enable
```
https://github.com/twbs/bootstrap/pull/25101/files/17a992f414d1c6963320cd1d877d205c85e8b82c#diff-d8ee409a461718bfb6240710c8c73382R11

Reply:
No worries, I remembered I had to do the same there when we switched to stylelint.